src/index.html

```diff
@@ -1,22 +1,399 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="page"
+  lang="en"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>MYBIKE</title>
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
+    <link
+      rel="shortcut icon"
+      href="./images/MyBIKE-32x32.png"
+      type="image/x-icon"
+    />
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__corpo">
+    <header
+      class="cabecalho menu"
+      id="cabecalho"
+    >
+      <div class="cabecalho__superior">
+        <a href="#cabecalho">
+          <img
+            src="./images/MyBIKE-logo-320.png"
+            alt="MyBIKE logo"
+            class="logo"
+          />
+        </a>
+
+        <div class="cabecalho__superior__icones">
```

Comment:
This nesting breaks BEM rules. Instead of `cabecalho__superior__icones`, use `cabecalho__icones`, since cabecalho is the block.